l10n_it_intrastat/models/account.py

```diff
@@ -0,0 +1,681 @@
+# Copyright 2019 Simone Rubino - Agile Business Group
+# Copyright 2024 Simone Rubino - Aion Tech
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_is_zero
+
+
+class AccountFiscalPosition(models.Model):
+    _inherit = "account.fiscal.position"
+
+    intrastat = fields.Boolean(string="Subject to Intrastat")
```

Comment:
Per compatibilità con https://github.com/OCA/intrastat-extrastat/blob/f66546a05b65c86882a94359015fd8fa66c11bb4/intrastat_base/models/account_fiscal_position.py#L12 e anche eventualmente altri, converrebbe rinominare i campi con prefisso `l10n_it`

Reply:
@eLBati 
> anche eventualmente altri

potresti essere più preciso? :)

Reply:
fatto in https://github.com/OCA/l10n-italy/compare/f14837433133fd6cbb582fb4cdbe7125d573f6a6..3b2444f2cd857be34d94e1a89fd18810a1de8ac5